defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-terraform_version: "0.6.12"
+terraform_version: "0.6.12"
```

Comment:
It looks like 0.6.16 makes some breaking changes to the AWS provider (aws_eip address can no longer be manually set). I know we use that provider/module so I want to be sure it won't break anything downstream.